request_test.go

```diff
@@ -80,3 +81,37 @@ func TestDescRegex(t *testing.T) {
 	assert.Equal(t, "[Ljava/lang/String;", results[0])
 	assert.Equal(t, "[I", results[1])
 }
+
+func TestToMapStringString(t *testing.T) {
```

Comment:
maybe u can create an issue for this pr, #1113 for example. And then, u can rename this test func name to TestIssue1113.